python/test/test_usability.py

```diff
@@ -113,3 +113,17 @@ def test_string_instead_of_tuple_for_margin_key():
     )
     counts.summarize()
     # counts.release().collect()
+
+
+def test_data_loader_error_is_human_readable():
+    pytest.importorskip("polars")
```

Comment:
Drop this line? I think you want to check there is nice behavior, whether or not polars is installed.

Reply:
The type check attempts to import Polars, which fails. I added this to get CI back on track.